### DISCUSSION TITLE
SO 2.4 - Zeek/Suricata Workers

### DISCUSSION THREAD

**Discussion by sjnhawk** [Security-Onion-Solutions/securityonion#11182] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Fresh install of SO 2.4.10 Standalone.

For Suricata and Zeek worker threads, I was hoping to get some clarification on what the values represent in each box (Current Grid Value/XX_standalone).

This is what I've got following a fresh install:

Suricata:
'Current Grid Value'=1
'XX_standalone'=3

Zeek:
'Current Grid Value'=0
'XX_standalone'=3

Is the '3' under my standalone instance what is currently assigned for each? If that is the case, do I make the change under "Current Grid Value"?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/67609108/0b47ac42-8d26-4d5d-baa6-72d54749a531)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment, marked as answer]
"Current Grid Value" is the value for the entire grid. Say you had 1000 sensors of all the same hardware and wanted them all to have 4 workers you would place it there. You can then override that for an individual minion with the minions dropdown.
